client/src/components/FormAction/FormAction.scss

```diff
@@ -41,7 +41,9 @@
 
 // Button icons
 .btn[class*="font-icon-"]:before,
-.dropdown-item[class*="font-icon-"]:before {
+.btn span[class*="font-icon-"]:before,
+.dropdown-item[class*="font-icon-"]:before,
+.dropdown-item span[class*="font-icon-"]:before  {
```

Comment:
This and similar changes are to support having the `font-icon-*` class on a child span, but also retain support for the icon to be on the specific element for anyone who's relying on that.